### DISCUSSION TITLE
Support for Backlinks

### DISCUSSION THREAD

**Discussion by stoweboyd** [alangrainger/share-note#51] [Vote for upcoming features]
When I upload a shared note with 'Backlinks in document' toggled on Share Note does upload and render some of the information associated with backlinks, but the headings that name the files backlinked are not clickable. The text is clickable for the section of the referenced link, but not the file headings. Likewise, all the backlinks controls are inactive.

Most important would be to make the file headings clickable.

![backlinks](https://github.com/alangrainger/share-note/assets/3037422/aa1e4350-b835-483e-89fd-6942432d1579)

**Comment by alangrainger** [comment]
This is now added in v1.1.2 :)

Demo:

https://share.note.sx/1wq529kt#nu4s71uEBB7mTWO7/eG3wF8xQqM1P0LHn3kLKT88vYM
